examples/dt/perfscale/scalelab/README.md

```diff
@@ -0,0 +1,34 @@
+# OpenStack Scalelab Baremetal
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/12c57baeca4ae33dd30a7707d330eb094309b4cd) on Dec 6th, 2023**
```

Comment:
Please use the commit here that corresponds to the OpenStack operator version you used in testing this patch.  I'm assuming you're not using the one listed from 2023, as that would be way out of date.